### PR TITLE
 [12.0][MIG] - account_invoice_accrual migration to 12.0

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: account_invoice_accrual/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "depends": [
+        "account",
+        "account_reversal",
+        "account_cutoff_accrual_base",
+    ],
+    "data": [
+        "views/account_invoice_view.xml",
+        "wizard/account_move_accrue_view.xml",
+    ],
+    "demo": [],
```
Comment (MeetKD) [inline account_invoice_accrual/__manifest__.py:19]:
```suggestion
```

File: account_invoice_accrual/wizard/account_move_accrue_view.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <record id="view_account_move_accrue" model="ir.ui.view">
+            <field name="name">account.move.accrue.form</field>
+            <field name="model">account.move.accrue</field>
+            <field name="arch" type="xml">
+                <form string="Create accrual journal entries" version="7.0">
```
Comment (MeetKD) [inline account_invoice_accrual/wizard/account_move_accrue_view.xml:7]:
```suggestion
                <form string="Create accrual journal entries">
```

**Comment by sbidoul** [comment]
@sbejaoui would you have a moment to look at this PR ? It would be nice to have it merged before the big migration to 14 starts. cc/ @alexis-via

**Comment by sbejaoui** [comment]
> @sbejaoui would you have a moment to look at this PR ? It would be nice to have it merged before the big migration to 14 starts. cc/ @alexis-via

This PR is blocked due to the dependency of `account_invoice_accrual` to `account_reversal` which was abandon and all the PRs are closed. 

A new module took place [account_document_reversal](https://github.com/OCA/account-financial-tools/tree/12.0/account_document_reversal). I couldn't read all the discussions about this move. But I believe it's better to get rid of this dependency if the standard reverse feature do the job. Otherwise we need to adapt `account_invoice_accrual` to the new module.

**Pull request closed without merging**